CHANGELOG.md

```diff
@@ -0,0 +1,49 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+
+## [Unreleased]
+
+## [0.2.1] - 2021-01-02
```

Comment:
Let's remove release notes for new version (I think it's nice to have releases not mixed with PR's that address different concerns)

Reply:
Done